### PR TITLE
Auto fix for license_overspecified

### PR DESCRIPTION
**Pull request by Arishamays1**
Autofix for `license_overspecified` in `check_completeness.py`

### CODE REVIEW

File: tests/test_completeness.py [hunk 1]
```diff
@@ -680,3 +680,23 @@ def test_wrong_output_script_key_bad(base_yaml):
     lint_check = "wrong_output_script_key"
     messages = check(lint_check, yaml_str)
     assert len(messages) == 1
+
+
+def license_file_overspecified_good(base_yaml):
```
Comment (marcoesters) [inline tests/test_completeness.py:685]: This test doesn't do what the docstring says it does. It only checks that there are no messages.

A better test would be to check the recipe before and after to ensure that the correct fix was applied (i.e., `license_url` was removed).

Comment (Arishamays1) [reply tests/test_completeness.py]: Sorry about the redundant code! Removed!

**Review by schuylermartin45 — APPROVED** [review]
Looks good. Welcome to the team, trooper!

salute

**Comment by Arishamays1** [comment]
> Looks good. Welcome to the team, trooper!
> 
> salute [ salute ](https://camo.githubusercontent.com/914d4968606fe168c8beebb5b027cc40b56fce8ba04f54bfda46ef21d08b6d1e/68747470733a2f2f6d656469612e74656e6f722e636f6d2f70514b707559386657566741414141432f7363686f6f6c2d6f662d726f636b2d6a61636b2d626c61636b2e676966) [ ](https://camo.githubusercontent.com/914d4968606fe168c8beebb5b027cc40b56fce8ba04f54bfda46ef21d08b6d1e/68747470733a2f2f6d656469612e74656e6f722e636f6d2f70514b707559386657566741414141432f7363686f6f6c2d6f662d726f636b2d6a61636b2d626c61636b2e676966)

Roger That!

**Pull request merged**